doc/api/events.md

```diff
@@ -574,6 +574,15 @@ to indicate an unlimited number of listeners.
 
 Returns a reference to the `EventEmitter`, so that calls can be chained.
 
+### emitter.rawListeners(eventName)
+<!-- YAML
+added: REPLACEME
+-->
+- `eventName` {any}
+
+Returns a copy of the array of listeners for the event named `eventName`,
+including any wrappers (such as those created by `.once`).
```

Comment:
The doc does not explain what a wrapper is in this context, and _how_ it is wrapped.

Reply:
Any thoughts on how we should document this or whether we even should? I'm not certain we want to necessarily *encourage* the usage. If someone needs it they'll know and otherwise we might want it to be a bit obscure, haha? Not sure...

If anyone has any thoughts, please chime in.

Reply:
If it's documented, it needs to be documented in full. Otherwise it doesn't. I'm fine either way.

Reply:
Do we have precedent for including a publicly exposed function that's not documented? Just trying to figure out the direction we would best go here. This isn't something we want to encourage being used but it's a step better than the current situation of reaching into `_events`.

Reply:
I'd rather this get documented fully.